ruhr-universitat-bochum-lehrstuhl-fur-industrial-sales-and-service-engineering.csl

```diff
@@ -102,10 +103,7 @@
     <choose>
       <if type="article-magazine article-newspaper article-journal bill book broadcast chapter entry-dictionary figure graphic interview legislation legal_case map motion_picture musical_score pamphlet paper-conference patent personal_communication report review review-book song speech thesis treaty dataset" match="any"/>
       <else>
-        <date variable="issued">
-          <date-part name="month" form="long" prefix=", "/>
-          <date-part name="day" form="ordinal" prefix=" "/>
-        </date>
+        <date variable="issued" form="text" prefix=" " />
```

Comment:
you're aware that this will repeat the year if no other date part is present? (Unfortunately impossible to test for that at this time)

Reply:
Yes, I'm aware of that. This is on purpose.